### ISSUE TITLE
Export crash

### ISSUE THREAD

**Issue by maxx53300** [OpenShot/openshot-qt#2104]
Hello

I tried to export my project but all the time the software crash no matter what resolution, codec or any other setting parameter i use

**System Details (please complete the following information):**
 - Operating System / Distro: Windows 7 pro
 - OpenShot Version 2.4.2

When you export your project on video format

Export my project to the video

**Screenshots**
![sans titre 2](https://user-images.githubusercontent.com/43218243/45447069-cfc34800-b6ce-11e8-930b-91ac6b843f5d.jpg)

**Logs**

[openshot-qt.log](https://github.com/OpenShot/openshot-qt/files/2376520/openshot-qt.log)

**Comment by BrackoNe** [comment]
I have same problem! I have figured out that Fade effect makes app crash, but there is something else as well. Think Transform makes crash too.

**Comment by BrackoNe** [comment]
OK, Channel Mapping was changed from -1 to 9. When I have back to -1 it works.

**Comment by BrackoNe** [comment]
One more thing you can check.
As you can see it crashed @ 13.8%
**Example:**
If video is 1:15 long, convert it to seconds and that is 75 seconds. You can open calculator and do: 75 x 0.138 = 10.35. That means to check 10.35 sec of your video. Issue should be in that (key) frame. That's the way how I located my issue.

**Comment by peanutbutterandcrackers** [comment]
@BrackoNe - Pretty interesting way of checking it. May I ask what exactly was the problem, in your case, with the (key) frame in question?

**Comment by BrackoNe** [comment]
> @BrackoNe - Pretty interesting way of checking it. May I ask what exactly was the problem, in your case, with the (key) frame in question?

Hi! Thanks! In my case it crashed every single time @33.2%. I have done this math and I have located that I have unintentionally changed Channel Mapping from -1 to 9. I do not even know what is this option.
Also It crashed few times at 22% - there was fade, but I have changed some fade options (not sure which options right now), and I have removed whole fade effect - it worked!

I have Windows 10.

Thanks,
Nemanja
